### PR TITLE
Add support for `@Links`

### PR DESCRIPTION
**Pull request by dobromir-hristov**
Bug/issue #, if applicable: 98312171

## Summary

Adds support for the new `@Links` directive from DocC - [Link to Proposal](https://forums.swift.org/t/highlighting-documentation-pages-outside-of-topics-sections-in-swift-docc-links-directive/59919)

image
image

**Note**
This is based on early RenderJSON drafts, so things might change.

## Dependencies

https://github.com/apple/swift-docc-render/pull/419
DocC PR

## Testing

[example_links.json.zip](https://github.com/dobromir-hristov/swift-docc-render/files/9452269/example_links.json.zip)

Use the provided minimal JSON 

Steps:
1. Navigate to the page
2. Assert it renders two grids of topics

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [x] Added tests
- [x] Ran `npm test`, and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

File: src/components/ContentNode/LinksBlock.vue [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      type: Array,
+      required: true,
+    },
+    blockStyle: {
+      type: String,
+      default: TopicStyles.compactGrid,
+    },
+  },
+  computed: {
+    isListList: ({ blockStyle }) => blockStyle === TopicStyles.list,
```
Comment (marinaaisa) [inline src/components/ContentNode/LinksBlock.vue]: I don't understand the name decision behind `isListList` 😄

**Comment by marinaaisa** [comment]
Tested and looks good! I had a minor comment.

**Review by marinaaisa — APPROVED** [review]

File: src/components/ContentNode/LinksBlock.vue (hunk 1 above)
Comment (dobromir-hristov) [reply src/components/ContentNode/LinksBlock.vue]: fixed

**Comment by dobromir-hristov** [comment]
Closed in favour of https://github.com/apple/swift-docc-render/pull/436

**Pull request closed without merging**